static/change-root-profile.sh

```diff
@@ -44,7 +44,8 @@ ROOT-PROFILE
 # Add Aliases
 {
 echo "alias nextcloud_occ='sudo -u www-data php $NCPATH/occ'"
-echo "alias run_update_nextcloud='bash $SCRIPTS/update.sh'"
+echo "alias nextcloud_update='bash $SCRIPTS/update.sh'"
```

Comment:
the reason why it's called *run*_nextcloud_update is because if you TAB the alias you might make a mistake and run the update instead of nextcloud_occ.

Reply:
maybe would run_nextcloud_update be okay?

Reply:
Hmm, it's been out there for a while now. I'm voting no. People may have it in their current crontabs and such. Already got emails about users that do.

Reply:
It is reverted.